src/test/regress/citus_tests/common.py

```diff
@@ -49,15 +62,17 @@ def add_settings(abs_data_path, settings):
             conf_file.write(setting)
 
 
-def create_role(pg_path, port, node_ports, user_name):
-    for port in node_ports:
```

Comment:
What is going on here? Was the `port` argument of the function unnecessary?
In that case what is the `coordinator_port` for?

Reply:
yeah it was not actually used. Removed the argument and changed its usage now.